### PR TITLE
Feature: dynamic welcome modal

### PR DESCRIPTION
**Pull request by sorodrigo**
This PR includes the following:
- [x] New redux store reducer/actions `modal`.
- [x] New `.env` variable WELCOME_MODAL_COOKIE_KEY
- [x] Loads dynamic content in welcome modal.
- [x] Style the welcome modal loaded content.
- [x] Test the complete behaviour with a  **real** cookie.

### CODE REVIEW

File: app/src/actions/modal.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import {
```
Comment (tiagojsag) [inline app/src/actions/modal.js]: let's rename this file to welcomeModal.js
This project has a lot of modal, and us-in-6-months will get confused by the simplistic name here.

File: app/src/components/Map/WelcomeModal.jsx [hunk 2]
```diff
@@ -1,17 +1,27 @@
+/* eslint-disable react/no-danger */
 import React, { Component } from 'react';
+import styles from 'styles/components/map/c-welcome-modal.scss';
 
-// TODO
 class WelcomeModal extends Component {
 
+  constructor(props) {
```
Comment (tiagojsag) [inline app/src/components/Map/WelcomeModal.jsx]: AFAIK this is redundant

File: README.md [hunk 3]
```diff
@@ -145,3 +145,7 @@ If set, should point to the local workspace to be loaded
 #### DISABLE_WELCOME_MODAL
 
 Disable welcome modal. Typically enabled in a dev environment, disabled in prod
+
```
Comment (tiagojsag) [inline README.md]: extra kudos for updating the docs ;)

File: app/src/actions/modal.js (hunk 1 above)
Comment (sorodrigo) [inline app/src/actions/modal.js]: I intended this file to host all modals eventually

File: app/src/components/App.jsx [hunk 4]
```diff
@@ -15,6 +15,20 @@ class App extends Component {
       }
     }
     this.props.getLoggedUser();
+
+    if (!DISABLE_WELCOME_MODAL) this.props.setWelcomeModalUrl();
```
Comment (nerik) [inline app/src/components/App.jsx]: I would highly suggest using explicit comparisons in that kind of situation, ie
```
DISABLE_WELCOME_MODAL !== true
```
You always get taken by surprise when smth is falsy or truthy unexpectedly.

Comment (nerik) [inline app/src/components/App.jsx]: `DISABLE_WELCOME_MODAL === true`

**Pull request merged**